Snake/Snake.ino

```diff
@@ -62,35 +62,54 @@ void THE_END()
 {
   Serial.println("Vesztettél!!!");
 
+  for (int i = 0; i < kigyoHossza + 1; i++)
```

Comment:
Ha jól sejtem, itt villogtatni szeretnéd a kígyót, ahogy az eredeti játék is csinálta, szuper ötlet! Ahogy elnézem, itt ugyanazt a kódrészletet másoltad be egymás után négyszer. Ez ugyan működik, de ha esetleg át kellene írni, négyszer annyi munka lenne. Ha van időd, próbáld meg egy (for) ciklusba szervezni, hogy csak egyszer szerepeljenek ezek a sorok:
`lc.setLed(0, kigyo[i].x, kigyo[i].y, false);`
`lc.setLed(0, kigyo[i].x, kigyo[i].y, true);`